File: get-customer-dashboard.js

const getCustomer = async () => {
  const resp = await fetch(`https://telco.axiom-hasura.private-ddn.hasura.app/v1/rest/auth/Users?page%5Blimit%5D=10&page%5Boffset%5D=0&fields%5BUsers%5D=email%2Cid`);
  const data = await resp.json();
  console.log(`data: ${JSON.stringify(data)}`)
};


const main = async () => {
  await getCustomer();
}


main()

File: get_customer_dashboard.py
import requests
import json

# GraphQL endpoint
url = "https://telco-0fda1a9112.axiom-hasura.private-ddn.hasura.app/graphql"

# GraphQL query
query = """
query CustomerDashboard {
  customers(
    where: {customerId: {_eq: "1"}}
    limit: 10
  ) {
    customerId
    firstName
    creditCards {
      expiry
      maskCreditCard
      number
      customerId
      createdAt
      updatedAt
    }
    customerPlans {
      cellNumber
      customerPlanId
      startDate
      endDate
      plan {
        description
      }
      customerPlanDevices {
        device {
          brand
          model
          updatedAt
        }
      }
    }
    customerLinks {
      customerPreferences {
        behavioralData {
          lastAppLogin
          lastWebsiteVisit
        }
      }
    }
  }
}
"""

# Headers (assuming no auth token is needed; add if required)
headers = {
    "Content-Type": "application/json",
}

# Payload
payload = {
    "query": query
}

# Make the POST request
response = requests.post(url, headers=headers, json=payload)

# Check response and print
if response.status_code == 200:
    data = response.json()
    print(json.dumps(data, indent=2))
else:
    print(f"Error: {response.status_code} - {response.text}")